app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,42 @@
+        \nClick 🔧 when showing favorites to open the menu.
         \n
-        \nIf you hold your finger a few moments on an application or a folder, you will be offered to access its settings.
+        \nLong-hold an app or folder to access its settings.
     </string>
     <string name="help_notification_title">Notification</string>
     <string name="help_notification_text">
-        The notification allows to display a popup with favorites above other applications when you leave the home screen.
-        In most cases, it should work directly and without issues.
-        \n
-        \nHowever, in some special cases it may bring back to the home screen instead of being displayed over other applications.
-        \nIf you are facing this issue, there are two things you can try:
-        \n- Press the \"clear all\" button in Recent Apps
-        \n- Reboot your smartphone
-        \nAfter one of these actions, the problem should not appear again.
-        \nIf it persists, I am sorry but it seems this will not work for you :-/
-        \n
-        \nTo not be intrusive, the notification icon is transparent and low priority.
-        With some applications having even lower priorities, this may result in an empty space in the status bar.
-        \n
-        \nFinally, if you change the default launcher, you may still see the notification.
-        To hide it, you can open Discreet Launcher (no need to be the default launcher) and disable the notification in the settings.
+        Shows a popup of favorites above other apps when you leave the home screen.
     </string>
     <string name="help_list_update_title">List update</string>
     <string name="help_list_update_text">
-        The list of applications and their icons are refreshed only after some events
-        (on start, when an application is added or removed, if the icon pack is modified).
-        When this happens, a small popup message appears during a few moments.
-        \n
-        \nIf something does not feel right, you should try to refresh the applications list manually.
+        A popup message is shown when refreshing the app list and icons,
+        when starting, adding or removing apps, or if the icon pack is modified.
     </string>
     <string name="help_folders_title">Folders</string>
     <string name="help_folders_text">
-        To create/rename/remove folders and put applications inside, you need to use the folders organizer.
-        It is accessible using the menu button or by clicking on the title of an open folder.
-        \n
-        \nYou can create as many folders as you want to organize your applications.
-        These folders can also be added in favorites.
-        If a folder is removed, its content will be put back in the main applications list.
-        \n
-        \nAn application can be in only one folder at a time (or not in a folder).
-        However, you can have both an application and the folder containing this application in favorites.

```

Comment:
This line and the one just above were added following a user request to clarify folders behavior (that an app can be only in one folder at a time or no folder at all, and that it is possible to have an app in favorites + a folder containing the same app also in favorites).
Probably this could be simplified but maybe something about this should be kept.

Reply:
I think only what isn't intuitive or obvious should be explained.
It is hard to get an overview with all the extra stuff.